MODULE.bazel

```
@@ -1,8 +1,24 @@
 module(
-    name = "buildbuddy_toolchain",
-    version = "0.0.0",
+    name = "toolchains_buildbuddy",
+    version = "0.0.1",
     repo_name = "io_buildbuddy_buildbuddy_toolchain",
 )
 
 bazel_dep(name = "rules_cc", version = "0.0.9")
 bazel_dep(name = "platforms", version = "0.0.10")
+
+buildbuddy = use_extension("//:extensions.bzl", "buildbuddy")
+use_repo(buildbuddy, "buildbuddy_toolchain")
+
+register_toolchains(
+    "//toolchains/cc:ubuntu_gcc",
+    "//toolchains/cc:windows_msvc",
```

```suggestion
    "//toolchains/cc:all",
```
It's possible that aliases aren't supported with wildcards, in that case you could reference the unaliased package here.

---

Yeah wild card does not work. Gona use the alias targets instead.